x-pack/plugins/monitoring/kibana.json

```diff
@@ -3,7 +3,7 @@
   "version": "8.0.0",
   "kibanaVersion": "kibana",
   "owner": {
-    "owner": "Stack Monitoring",
+    "name": "Stack Monitoring",
     "githubTeam": "stack-monitoring-ui"
```

Comment:
We also have stack-monitoring but this one seems more "maintained" so let's keep it. FYI @ravikesarwani you're missing from @elastic/stack-monitoring-ui so I'll add you.